.eslintrc.json

```diff
@@ -0,0 +1,29 @@
+{
+    "extends": [
+        "airbnb"
+    ],
+    "plugins": [
+        "react"
+    ],
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true,
+            "impliedStrict": true
+        },
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
+        "quotes": 0,
```

Comment:
What does the 0 symbolise here. Do you think it would be better if we specified to use either single or double quotes for consistency and as a convention for us as a team?

Reply:
That is a rule specified by eslint Vicky.
0 is a number I can't put it in quotes because that will make it a string

Reply:
In addition to @FahdJamy's point, the `eslint` config file has two keys for when a setting is `off`. `0` or `off`. The convention at Andela seems to be using `0`, deducing from [this gist](https://gist.github.com/sa-adebayo/fe846cfb1c8c16660a7343aa94d21d65).